Server/server.js

```diff
@@ -39,15 +39,12 @@ app.get('/api/reviews', (req, res) => {
       res.send(results);
     }
   });
-  // .then(function(results) {
-  //   res.send(results);
-  // })
 })
 
 app.get('/:id', (req, res) => {
   ReviewsController.getById(req.params.id, (err, result) => {
     if (err) {
-      console.log('ERROR')
+      console.log('ERROR');
       throw new Error();
```

Comment:
I know this isn't your part of the code, but is just throwing a new error what you want to do here?

Reply:
Very good point.  I'm still trying to decide whether to keep this type of error handling.  It's going to depend on what the front end is doing.  Will take a look after prezzies.